src/array.cr

```diff
@@ -671,15 +671,38 @@ class Array(T)
     self
   end
 
-  # Removes all items from `self` that are equal to *obj*.
+  # Removes all items by default from `self` that are equal to *obj*.
   #
   # ```
   # a = ["a", "b", "b", "b", "c"]
   # a.delete("b")
   # a # => ["a", "c"]
   # ```
-  def delete(obj)
-    reject! { |e| e == obj } != nil
+  #
+  # :only option removes only specified number of items from `self` that are equal to *obj*.
```

Comment:
They're really not symbols, use `*only*`, see http://crystal-lang.org/docs/conventions/documenting_code.html